.github/workflows/_publish_container.yaml

```diff
@@ -85,26 +92,26 @@ jobs:
             docker buildx imagetools create --tag $tag ${{ steps.get-manifests.outputs.manifests }}
```

Comment:
Does this work instead?

`docker buildx imagetools create $(echo "${{ inputs.SOURCE_IMAGE }}" | tr '\n' ' ') --tag ${{ inputs.TARGET_IMAGE }}`

Reply:
That won't work because:
1. `inputs.TARGET_IMAGE` is just the part before `:`)
2. `inputs.TARGET_IMAGE` may contain multiple lines of tags using the docker-metadata action format. For example, see https://github.com/NVIDIA/JAX-Toolbox/blob/5ae2e7913558ff134e17c592ea40f0aadc1f04d2/.github/workflows/nightly-jax-build.yaml#L66C12-L66C12

Also, the reason why I use the `manifests` step to pick up the SHAs is to get ride of an 'unknown' platform that gets added by buildx for every image. It is largely cosmetic but could still be confusing for end users.